src/vcf/meta_entry.cpp

```diff
@@ -214,6 +214,13 @@ namespace ebi
             value["Type"] != "String") {
             throw new MetaSectionError{entry.line, "INFO metadata Type is not a Integer, Float, Flag, Character or String"};
         }
+
+        // Check for predefined tag AA
+        if (value["ID"] == "AA") {
+            if (value["Type"] != "String") {
```

Comment:
Separated this out from the upper if block instead of using `&&` operator. I think its better because we might need to add 2 or more checks for other tags and hence the separation avoids us from making the outer if block too length or complicated (using a lot of logical operators).